toolchain/mfc/run/case_dicts.py

```diff
@@ -1,5 +1,45 @@
 from ..state import ARG
 
+# Currently doing this manually -- the first automated solution I came up with involved grep and awk, but it seemed quite brittle.
+INTEGERS = {
+    "m", "n", "p", 'thermal', 't_step_old', 't_step_start', 't_step_stop',
+    't_step_save', 'num_patches', 'model_eqns', 'num_fluids', "weno_order",
+    "precision", "relax_model", 'perturb_flow_fluid', 'perturb_sph_fluid',
+    'dist_type', 'R0_type', 'num_ibs', 't_step_print', 'time_stepper',
+    'riemann_solver', 'wave_speeds', 'avg_state', 'fd_order', 'num_probes',
+    'bubble_model', 'num_mono', 'num_integrals', 'format', 'flux_lim',
+}
+REALS = {
+    "pref", "Web", "poly_sigma", 'x_domain%beg', 'x_domain%end', 'y_domain%beg',
+    'y_domain%end', 'dt', "nb", "rhoref", "R0ref", "Re_inv", "Ca",
+    'perturb_flow_mag', 'sigR', 'sigV', 'rhoRV', "palpha_eps", "ptgalpha_eps",
+    'pi_fac', 'weno_eps', 'schlieren_alpha', 'fluid_rho',
+}
+LOGICALS = {
+    "hypoelasticity", "cyl_coord", "parallel_io", "run_time_info", "bubbles",
+    "polytropic", "polydisperse", "file_per_process", "relax", "mpp_lim", 
+    "adv_alphan",  "adv_n", 'old_grid', 'old_ic', 'instability_wave',
+    'perturb_flow', 'perturb_sph', 'qbmm', 'ib', 'mapped_weno', 'mp_weno',
+    'weno_avg', 'weno_Re_flux', 'prim_vars_wrt', 'alt_soundspeed', 
+    'null_weights', 'probe_wrt', 'Monopole', 'integral_wrt', 'cu_mpi', 
+    'adap_dt', 'schlieren_wrt', 'alpha_rho_wrt', 'rho_wrt', 'mom_wrt', 
+    'vel_wrt', 'flux_wrt', 'E_wrt', 'pres_wrt', 'alpha_wrt', 'kappa_wrt',
+    'gamma_wrt', 'heat_ratio_wrt', 'pi_inf_wrt', 'pres_inf_wrt',
+    'cons_vars_wrt', 'c_wrt', 'omega_wrt', 'qm_wrt', 'vel_profile',
+    'mixture_err',
+}
+STRINGS = {"case_dir"}
```

Comment:
@AiredaleDev I think you can just initialize these as empty now.

Reply:
This actually prompted me to refactor one more time since I found a way I could easily regex-replace what I currently have into something simpler.